packages/core/src/browser/dialogs.ts

```diff
@@ -166,8 +166,10 @@ export abstract class AbstractDialog<T> extends BaseWidget {
         container.classList.add('dialogBlock');
         if (props.maxWidth === undefined) {
             container.setAttribute('style', 'max-width: none');
-        } else {
+        } else if (props.maxWidth < 400) {
             container.setAttribute('style', `max-width: ${props.maxWidth}px; min-width: 0px`);
+        } else {
+            container.setAttribute('style', `max-width: ${props.maxWidth}px`);
```

Comment:
Question: Why would we want to remove the `min-width` style in case `max-width` is `>= 400`?

Reply:
The dialogs already have a minimum-width of 400 px from here:
From my understanding, the only case where we would need to override this `min-width: 400px` and set it to `0px` would be when we have a `max-width < 400`. In the other cases, we should keep the value from `.dialogBlock` and avoid overriding it which is why I removed the `min-width` style. Please tell me if I am wrong.
https://github.com/eclipse-theia/theia/blob/803efc994ef71352d36d4dfdc6fb87183cb9b6b7/packages/core/src/browser/style/dialog.css#L32-L40

Reply:
I see, I didn't know about that. Makes sense now 👍